source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,176 @@
+- Press ESC to clear channel or Direct Message selections.
+
+Multi-selected channels and Direct Messages move together as a group in the order they originally appeared.
+
+Dragging and dropping selections
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Using the Mattermost Web or Desktop App, drag selected channels and/or Direct Messages between or within categories. 
+
+    .. image:: ../../images/multi-select-drag.gif
+
+Moving selections
+^^^^^^^^^^^^^^^^^
+
+In addition to selecting and dragging, you can also specify a destination for selected channels and/or Direct Messages using the **Move to** option under the **Channel options** icon in the sidebar.  
+
+    .. image:: ../../images/multi-select-move.gif
+
+Muting and unmuting categories
+------------------------------
+
+When you mute or unmute a category, all channels within that category are also muted or unmuted. 
+
+Select the **Category options** icon in the sidebar, then select **Mute Category**.
+
+Once a category is muted:
+
+- Email, desktop, and push notifications are disabled for all channels in the category.
+- A mute icon displays next to each channel name in the category.
+- The category and all of its channels are greyed out in the left-hand sidebar. Channels in the category aren't marked as unread unless you’re mentioned directly.
+- You can selectively unmute specific channels within the category.
+
+To unmute the category, select the **Category options** icon in the sidebar, then select **Unmute Category**.
+
+    .. image:: ../../images/mute-categories.gif
+
+Sorting category channels
+-------------------------
+
+Using the Mattermost Web or Desktop App, you can control how channels are sorted for each category. Sort category channels manually (by dragging and dropping), alphabetically, or by recent conversations first.
+
+Select the **Category options** icon in the sidebar, then select **Sort** and choose from **Alphabetically**, **Recent Activity**, or **Manually**.
+
+    .. image:: ../../images/sort-categories.gif
+
+Grouping unread messages
+------------------------
+
+Catch up on all your unread channels in one place at the top of your sidebar with a one-click **Unreads** category.
+
+Go to **Main Menu > Account Settings > Sidebar**, set **Group unread channels separately** to **On**, then select **Save**.
+
+- When this setting is enabled, all unread messages appear only in the **Unreads** category.
+- When this setting is disabled, all unread messages appear within their respective categories and channels.
+
+When enabled, the **Unreads** category is organized as follows:
+
+- Unread messages that contain mentions are sorted to the top by most recent activity.
+- Unread messages that do not contain mentions are sorted by most recent and appear directly below unread messages containing mentions.
+- Muted channels that contain mentions are sorted by most recent and appear directly below unread messages without mentions.
+
+    .. image:: ../../images/unreads.gif
```

Comment:
In this gif can we start with showing the unread filter being toggled, then go to account settings and enable unreads section?